### ISSUE TITLE
Enable info page option in the settings

### ISSUE THREAD

**Issue by nikidev** [FMCorz/moodle-block_xp#93]
Hello

While I was playing with the global settings of the plugin I found that the "Enable info page" option is not working. When I try to turn it off (untick the checkbox) the Info button is still displaying on the block.
![image](https://user-images.githubusercontent.com/6606146/53580487-6b187e80-3b84-11e9-977d-5dfb89d4d56f.png)

![image](https://user-images.githubusercontent.com/6606146/53580532-82f00280-3b84-11e9-853a-7df066e799d5.png)

The same situation is with the ladder option.

**Comment by FMCorz** [comment]
Thanks for the report @nikidev. We've tested with the latest release on both themes boost and clean and the block hid the links as expected. I'm afraid you have been changing the default values, not the value for the block itself. Once the block has been added, the default values no longer have an impact and the block settings should be used. Thanks!

**Comment by nikidev** [comment]
Alright ! I got your point ! It works in that way. Thanks !  👍   :)

I tried also to set moodle get_config() function for that setting in that place of the code for that checbox  to make it to work as I expect. I wanted to remove  Info button to all level up blocks at once. And it worked in that way too.  But you probably made the settings like that because you  want to take custom control  on the navigation of every instance of the block on every course.  :)
